app/src/main/java/ch/hikemate/app/model/authentication/FirebaseAuthRepository.kt

```diff
@@ -61,6 +65,8 @@ class FirebaseAuthRepository {
                 request = request, // Send the request we built
                 context = context // Provide the context for the request
                 )
+        if (result.credential.type != GoogleIdTokenCredential.TYPE_GOOGLE_ID_TOKEN_CREDENTIAL)
+            throw Exception("Invalid credential type")
```

Comment:
Could you document why you did this here ? Was there an issue without this check ?

Reply:
Done :)